templates/dashboard/settings/withdraw-settings.php

```diff
@@ -1,136 +1,154 @@
+
+			foreach ( $tutor_withdrawal_methods as $method_id => $method ) {
+				$form_fields = tutor_utils()->avalue_dot( 'form_fields', $method );
+
+				$method_values                                = get_user_meta( get_current_user_id(), '_tutor_withdraw_method_data_' . $method_id, true );
+				$method_values                                = maybe_unserialize( $method_values );
+				! is_array( $method_values ) ? $method_values = array() : 0;
+				?>
+
+					<div data-withdraw-form="<?php echo esc_attr( $method_id ); ?>" class="tutor-row withdraw-method-form" style="<?php echo esc_attr( $old_method_key != $method_id ? 'display: none;' : '' ); ?>">
+					<?php
+					do_action( "tutor_withdraw_set_account_{$method_id}_before" );
+
+					$field_count = tutor_utils()->count( $form_fields );
+					if ( $field_count ) {
+						foreach ( $form_fields as $field_name => $field ) {
+							?>
+								<div class="<?php echo esc_attr( $field_count ) > 1 ? 'tutor-col-12 tutor-col-sm-6' : 'tutor-col-12'; ?> tutor-mb-32">
+								<?php
+								if ( ! empty( $field['label'] ) ) {
+									$markup = "<label class='tutor-form-label tutor-color-muted' for='field_{$method_id}_$field_name'>
+                                                " . htmlspecialchars( $field['label'] ) . '
+                                            </label>';
+
+									echo wp_kses(
+										$markup,
+										array(
+											'label' => array(
+												'class' => true,
+												'for'   => true,
+											),
+										)
+									);
+								}
+
+								$passing_data = apply_filters(
+									'tutor_withdraw_account_field_type_data',
+									array(
+										'method_id'  => $method_id,
+										'method'     => $method,
+										'field_name' => $field_name,
+										'field'      => $field,
+										'old_value'  => null,
+									)
+								);
+
+								$old_value = tutor_utils()->avalue_dot( $field_name . '.value', $saved_account );
+								if ( $old_value ) {
+									$passing_data['old_value'] = $old_value;
+								} elseif ( isset( $method_values[ $field_name ], $method_values[ $field_name ]['value'] ) ) {
+									$passing_data['old_value'] = $method_values[ $field_name ]['value'];
+									$old_value                 = $passing_data['old_value'];
+								}
+
+								if ( in_array( $field['type'], array( 'text', 'number', 'email' ) ) ) {
+									?>
+											<input class="tutor-form-control tutor-mt-4" type="<?php echo esc_attr( $field['type'] ); ?>" name="withdraw_method_field[<?php echo esc_attr( $method_id ); ?>][<?php echo esc_attr( $field_name ); ?>]" value="<?php echo esc_attr( $old_value ); ?>" >
+										<?php
+								} elseif ( 'textarea' == $field['type'] ) {
+									?>
+											<textarea class="tutor-form-control tutor-mt-4" name="withdraw_method_field[<?php echo esc_attr( $method_id ); ?>][<?php echo esc_attr( $field_name ); ?>]">
+											<?php echo wp_kses_post( $old_value ); ?>
```

Comment:
Please use esc_textarea instead of wp_kses_post

Reply:
Done